### PR TITLE
Adjust letter spacing for Title, smoothen Search animation

### PR DESCRIPTION
**Pull request by Shriram-Balaji**
Hola! 

I noticed that the Title component had issues with Letter spacing which made it look slightly cramped for space. I've adjusted it to make it slightly more readable. Also I've made the Search animation 0.25s longer just to smoothen it a bit. 

**Before**

before1
before2

**After**

after1
after2

### CODE REVIEW

File: src/Components/Search.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const Wrapper = styled.div`
     display: flex;
     width: ${remcalc(300)};
     position: relative;
-    transition: all 0.25s ease-in-out;
+    transition: all 0.5s ease-in-out;
```
Comment (hosmelq) [inline src/Components/Search.js]: I think this is much time for the animation.

Comment (Shriram-Balaji) [reply src/Components/Search.js]: Okay, I'll probably make it 0.35s. To my eyes, 0.25s felt too quick, that was the idea behind increasing it.

Comment (SaraVieira) [reply src/Components/Search.js]: Are you okay with 0.35 @hosmelq?

Comment (hosmelq) [reply src/Components/Search.js]: I tried it and that speed seems good to me.

Comment (SaraVieira) [reply src/Components/Search.js]: Awesome! Merging it 😃

**Comment by SaraVieira** [comment]
I will upload this tomorrow, it's already one am here :(

**Pull request merged**